src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtensionInline.cs

```diff
@@ -0,0 +1,198 @@
+                        return false;
+                }
+            }
+
+            if(string.IsNullOrEmpty(type))
+            {
+                type = "content";
+            }
+
+            if (string.IsNullOrEmpty(src))
+            {
+                logError("source is a required attribute. Please ensure you have specified a source attribute.");
+            }
+            if (string.IsNullOrEmpty(alt) && type != "icon")
+            {
+                logError("alt-text is a required attribute. Please ensure you have specified an alt-text attribute.");
+            }
+            if ((string.IsNullOrEmpty(alt) && type != "icon") || string.IsNullOrEmpty(src))
+            {
+                return false;
+            }
+            htmlAttributes = new HtmlAttributes();
+            htmlAttributes.AddProperty("src", _context.GetLink(src, inline));
+
+            if (type == "icon")
+            {
+                htmlAttributes.AddProperty("role", "presentation");
+            } else
+            {
+                htmlAttributes.AddProperty("alt", alt);
+            }
+            var id = GetHtmlId(inline);
+            if(type == "complex") htmlAttributes.AddProperty("aria-describedby", id);
+
+            RenderDelegate = (renderer, obj) =>
+            {
+                var currentType = string.Empty;
+                var currentLightbox = string.Empty;
+                var currentBorderStr = string.Empty;
+                var currentBorder = true;
+                var currentLink = string.Empty;
+                if(!obj.Attributes.TryGetValue("type", out currentType))
+                {
+                    currentType = "content";
+                }
+                obj.Attributes.TryGetValue("lightbox", out currentLightbox); //it's okay if this is null
+                obj.Attributes.TryGetValue("border", out currentBorderStr); //it's okay if this is null
+                obj.Attributes.TryGetValue("link", out currentLink); //it's okay if this is null
+                if (!bool.TryParse(currentBorderStr, out currentBorder))
+                {
+                    if(currentType == "icon")
+                    {
+                        currentBorder = false;
+                    } else
+                    {
+                        currentBorder = true;
+                    }
+                }
+
+                if(!string.IsNullOrEmpty(currentLink))
+                {
+                    var linkHtmlAttributes = new HtmlAttributes();
+                    currentLink = _context.GetLink(currentLink, obj);
+                    linkHtmlAttributes.AddProperty("href", $"{currentLink}");
+                    renderer.Write("<a").WriteAttributes(linkHtmlAttributes).WriteLine(">");
+                } else if (!string.IsNullOrEmpty(currentLightbox))
+                {
+                    var lighboxHtmlAttributes = new HtmlAttributes();
```

Comment:
Instead of putting it into expected, fix the typo?
```suggestion
                    var lightboxHtmlAttributes = new HtmlAttributes();
```

Reply:
Oh my... my eyes got tired. Good call!